src/lib.rs

```diff
@@ -33,8 +33,14 @@ extern crate cfg_if;
 use core::mem::MaybeUninit;
 
 mod backends;
+#[allow(dead_code, reason = "not used in all backends")]
 mod error;
+#[allow(dead_code, reason = "not used in all backends")]
+mod lazy;
+#[allow(dead_code, reason = "not used in all backends")]
 mod util;
+#[allow(dead_code, reason = "not used in all backends")]
+mod util_libc;
```

Comment:
We can't unconditionally include `util_libc` as it depends on `libc` which isn't used on all platforms.